pytest.ini

```diff
@@ -65,3 +65,4 @@ filterwarnings =
     ignore:unclosed \<socket:ResourceWarning
     # Pending removal from sunpy 7.0
     ignore:The QueryResponse class is deprecated
+    ignore:The toplevel space of the io subpackage was never
```

Comment:
Are we sure that we're importing from file_tools everywhere and not the top level namespace? This would hide it if we are.

Reply:
Err, good question. I will do a check with this removed and find out.